rocket_twin/drivers/vertical_flying_rocket.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from cosapp.drivers import Driver, NonLinearSolver, RungeKutta
+from cosapp.recorders import DataFrameRecorder
+from cosapp.systems import System
+
+
+class VerticalFlyingRocket(Driver):
+    """Driver that simulates the vertical flight of a rocket.
+
+    Inputs
+    ------
+    name: string,
+        the name of the driver
+    w_out [kg/s]: float,
+        mass flow of fuel exiting the rocket tank
+    dt [s]: float,
+        integration time step
+    owner: System,
+        the system that owns the driver
+
+    Outputs
+    ------
+    """
+
+    def __init__(self, name: str, w_out, dt, owner: Optional["System"] = None, **kwargs):
```

Comment:
see above how to pass init, stop and includs

Reply:
Check